tests/Agent/IntegrationTests/SharedApplications/Common/MultiFunctionApplicationHelpers/NetStandardLibraries/LogInstrumentation/SerilogLoggingWebAdapter.cs

```diff
@@ -40,9 +40,12 @@ public void Warn(string message)
             var result = _client.GetStringAsync(_uriBase + "test?logLevel=WARN&message=" + message).Result;
         }
 
-        public void Error(string message)
+        public void Error(Exception exception)
         {
-            var result = _client.GetStringAsync(_uriBase + "test?logLevel=ERROR&message=" + message).Result;
+            // In this case we are not passing the exact Exception to the test app, just the message.
+            // The test app will create an Exception for us.
+            // As long as it has the same message and class with a stacktrace of any kind, its good for test.
+            var result = _client.GetStringAsync(_uriBase + "test?logLevel=ERROR&message=" + exception.Message).Result;
```

Comment:
I was actually thinking of removing all but info and error(exception) .  Having more than one level is not testing anything more in instrumentation, instead it is testing the logging framework itself.  if we can capture info level, we will capture any of the others.  The same goes for capturing exceptions, since all the levels take an exception in some form - using error just made the most sense thematically.  The logging tests are also way to large at this point, so adding more would be less than ideal.

Reply:
I don't see how we can test a log message with an exception, but no message with the current model (in this webapp, and the ConsoleMF api). In this case I was thinking we could just add another query parameter to the test app for exceptions. 

Regarding testing different log levels, I believe there is value in this, especially when we add some tests that verify the configured log level is respected. While I agree that testing all log levels may be exhaustive, there is little overhead (except possibly in maintaining the tests and building them initially).

I'm not saying we need a whole separate test harness (which would add more time), I was just saying it should be clear by reading the tests that we are testing an exception, and log events with/without messages.